From: Richard B Winters <rik@mmogp.com>
Date: Sun, 3 May 2015 20:49:44 -0400
Subject: Fix file locations in pcsd.service

 - The locations for the system configuration and the init 
   script needed updating for Debian

Change-Id: Ifa7600fc56a80a5631bbdb731a68e8f18b4d17e0
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 pcsd/pcsd.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/pcsd/pcsd.service
+++ b/pcsd/pcsd.service
@@ -2,9 +2,9 @@
 Description=PCS GUI and remote configuration interface
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/pcsd
+EnvironmentFile=-/etc/init/pcsd.conf
 WorkingDirectory=/var/lib/pcsd
-ExecStart=/usr/lib/pcsd/pcsd start
+ExecStart=/etc/init.d/pcsd start
 Type=notify
 NotifyAccess=all
 
